root/edit/search_macros.tt

```diff
@@ -19,6 +19,10 @@
     [%- sort_select_block = '<select name="order">
       <option value="desc"' _ selected_if_matches('desc', query.order, 1) _ '>' _ l('newest first') _ '</option>
       <option value="asc"'  _ selected_if_matches('asc', query.order)  _ '>' _ l('oldest first') _ '</option>
+      <option value="closed-desc"' _ selected_if_matches('closed-desc', query.order) _ '>' _ l('recently closed first')     _ '</option>
+      <option value="closed-asc"' _ selected_if_matches('closed-asc', query.order) _ '>' _ l('earliest closed first')     _ '</option>
+      <option value="voteclosing-asc"' _ selected_if_matches('voteclosing-asc', query.order) _ '>' _ l('closing sooner first')     _ '</option>
+      <option value="voteclosing-desc"' _ selected_if_matches('voteclosing-desc', query.order) _ '>' _ l('closing later first')     _ '</option>
```

Comment:
Labels should specify it is about _vote_ closing.

Reply:
I feel the difference for the user here is pretty much nonexistent, but it doesn't hurt, so I'll change it.